event-svc/src/event/ordering_task.rs

```diff
@@ -17,6 +21,19 @@ type EventCid = Cid;
 type PrevCid = Cid;
 
 const LOG_EVERY_N_ENTRIES: usize = 10_000;
+/// The max number of tasks we'll spawn when ordering events.
+/// Each stask is given a chunk of streams and wil order them
+const MAX_STREAM_PROCESSING_TASKS: usize = 16;
+/// The minimim number of streams each task will process. If there are many streams, we'll spawn
+/// STREAM_NUM / MAX_STREAM_PROCESSING_TASKS.
+const MIN_NUM_STREAMS_PER_BATCH: usize = 25;
```

Comment:
What is `STREAM_NUM`?

Reply:
oops, that comment makes no sense (and it looks like a const). it was supposed to explain how the batch for each task is determined. I'll fix it.